_data/affiliations.yaml

```diff
@@ -136,3 +136,9 @@
   expose: false
   type: infrastructure
   url: https://nels.bioinfo.no/
+- name: European Union
+  image_url: /images/funders/Flag_of_Europe.svg
```

Comment:
shouldn't it be the EC logo instead of the EU flag, as the doi is pointing to the EC?

E.g. https://ec.europa.eu/info/publications/standard-european-commission-logos_en

Reply:
You are right, I updated it :)